docs/enforcement/inline.asciidoc

```diff
@@ -0,0 +1,255 @@
+Introduction
+~~~~~~~~~~~~
+
+The inline enforcement is a very convenient method for performing access control on older network equipment that is not capable of doing VLAN enforcement or that is not supported by PacketFence.
+
+An important configuration parameter to have in mind when configuring inline enforcement is that the DNS reached by these users should be your actual production DNS server - which shouldn't be in the same broadcast domain as your inline users. The next section shows you how to configure the proper inline interface and it is in this section that you should refer to the proper production DNS.
+
+Inline enforcement uses `ipset` to mark nodes as registered, unregistered and isolated.
+It is also now possible to use multiple inline interfaces. A node registered on the first inline interface is marked with an IP:MAC tuple (for L2, only ip for L3), so when the node tries to register on an other inline interface, PacketFence detects that the node is already registered on the first inline network.
+It is also possible to enable inline.should_reauth_on_vlan_change to force users to reauthenticate when they change inline network.
+
+By default the inline traffic is forwarded through the management network interface but it is possible to specify another one by adding in pf.conf the option interfaceSNAT in inline section. It is a comma delimited list of network interfaces like eth0,eth1.2. It's also possible to specify a network that will be routed instead of using NAT by adding in `conf/networks.conf` an option nat=no under one or more network sections (take care of the routing table of the PacketFence server).
```

Comment:
by adding the option interfaceSNAT in the inline section of pf.conf

Reply:
(don't forget to adjust the routing table of the PacketFence server)

Reply:
Routing? Don't we do that automatically now? if not, can you write a small blurb for me?

Reply:
We don't route by default, we NAT.

Not sure how to define the iptables rules for routing instead of NATing since I never did it.

Probably our inline king @fdurand can help with this?